Makefile.am

```diff
@@ -631,6 +631,14 @@ test_common_libeos_updater_test_common_la_SOURCES = \
 	test-common/utils.h \
 	$(NULL)
 
+tests_services_in = \
+	tests/services/org.freedesktop.Flatpak.SystemHelper.service.in \
+	$(NULL)
+
+systemdsystemunit_DATA += $(tests_services_in:%.in=%)
```

Comment:
Not sure the `systemdsystemunit_DATA` variable is the right one for this; that means this mock D-Bus service file will be installed system-wide in the systemd system unit directory.

Reply:
Ah right that was wrong on two levels: it's not a systemd unit and it shouldn't be installed. Pushed a fixup to put it in `noinst_DATA` and verified that it doesn't get installed but the tests still pass. Hopefully that's right; autotools are still a mystery to me

Reply:
Looks good, although I realise I have just merged the fixups unsquashed. :-(